dwave/system/testing.py

```diff
@@ -117,69 +127,211 @@ def __init__(self, broken_nodes=None, broken_edges=None,
+                'Flux biases for each working qubit, in normalized offset '
+                'units, formatted as a list for all qubits.',
+                'flux_drift_compensation':
+                'Activation of flux drift compensation, as a boolean flag.',
+                'h_gain_schedule':
+                "h-gain schedule formatted as a piecewise linear list of "
+                "floating-point pairs of 't' and 'g'.",
+                'initial_state':
+                'Initial states to use for a reverse-anneal request, as a list '
+                'of qubit index and state.',
+                'max_answers':
+                'Maximum number of answers to return.',
+                'num_reads':
+                'Number of states to read (answers to return), as a positive '
+                'integer.',
+                'num_spin_reversal_transforms':
+                'Number of spin-reversal transforms (gauge transformations) to '
+                'perform.',
+                'programming_thermalization':
+                'Time in microseconds to wait after programming the processor '
+                'in order for it to cool back to base temperature, as a '
+                'positive floating point number.',
+                'readout_thermalization':
+                'Time in microseconds to wait after each state is read from '
+                'the processor in order for it to cool back to base '
+                'temperature, as a positive floating point number.',
+                'reduce_intersample_correlation':
+                'Addition of pauses between samples, as a boolean flag.',
+                'reinitialize_state':
+                'Reapplication of the initial_state for every read in reverse '
+                'annealing, as a boolean flag.'},
+            'vfyc': False,
+            'anneal_offset_step': -0.0001500217998314891,
+            'anneal_offset_step_phi0': 1.4303846404537006e-05,
+            'annealing_time_range': [0.02, 83000.0],
+            'default_annealing_time': 20.0,
+            'default_programming_thermalization': 1000.0,
+            'default_readout_thermalization': 0.0,
+            'extended_j_range': [-2.0, 1.0],
+            'h_gain_schedule_range': [-3.0, 3.0],
+            'max_anneal_schedule_points': 50,
+            'max_h_gain_schedule_points': 20,
+            'num_reads_range': [1, 10000],
+            'per_qubit_coupling_range': [-18.0, 15.0],
+            'problem_run_duration_range': [0.0, 10000000.0],
+            'programming_thermalization_range': [0.0, 10000.0],
+            'readout_thermalization_range': [0.0, 10000.0],
+            'tags': [],
+            'category': 'qpu',
+            'quota_conversion_rate': 1}
         
     @dimod.bqm_structured
-    def sample(self, bqm, num_reads=1, flux_biases=[], **kwargs):
-        # we are altering the bqm if flux_biases given
+    def sample(self, bqm, **kwargs):
+        ''' dimod sampler for mocking DWaveSampler() interface.
```

Comment:
More consistent with the rest of the library to use `"""`. Also `Mock DWaveSampler().` probably covers it